ERCS/erc-7591.md

```diff
@@ -0,0 +1,40 @@
+---
+eip: 7591
```

Comment:
```suggestion
eip: 7617
```
Assigning next sequential EIP/ERC/RIP number.  7591 was already assigned.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Thanks! Will do. I think I missed that.